dist/gcc-compatible/EverCrypt_Hash.h

```diff
@@ -85,6 +85,24 @@ EverCrypt_Hash_Incremental_update(
   uint32_t len
 );
 
+void
+EverCrypt_Hash_Incremental_finish_sha3_256(
```

Comment:
why do we need to expose these functions? aren't they covered by `EverCrypt_Hash_Incremental_finish`?

Reply:
good catch, I neglected to add some `private` qualifier... fixed by my latest commit